### ISSUE TITLE
Support for symfony 5.3

### ISSUE THREAD

**Issue by joanna-bak-sourceability** [M6Web/ElasticsearchBundle#30]
Hello,

Could you add support for symfony 5.3?

Current error that we get after cache clear: 
```
Fatal error: Uncaught Symfony\Component\Debug\Exception\FatalThrowableError: Too few arguments to function Symfony\Component\Config\Definition\Builder\TreeBuilder::__construct(), 0 passed in /application/vendor/m6web/elasticsearch-bundle/src/M6Web/Bundle/ElasticsearchBundle/DependencyInjection/Configuration.php on line 20 and at least 1 expected in /application/vendor/symfony/config/Definition/Builder/TreeBuilder.php:26
```

**Comment by joanna-bak-sourceability** [comment]
Can we merge this? https://github.com/M6Web/ElasticsearchBundle/pull/31

**Comment by Oliboy50** [comment]
Yes, I think

we just have to wait for another M6Web organization member to approve/merge/release it

cc @b-viguier for example :)

**Comment by Oliboy50** [comment]
fixed with #31 

thanks @joanna-bak-sourceability
